Prefabs/Interactions/Controllables/SharedResources/Scripts/RotationalDrive.cs

```diff
@@ -52,12 +60,21 @@ public abstract class RotationalDrive : Drive<RotationalDriveFacade, RotationalD
         /// The current actual rotational value of the drive.
         /// </summary>
         protected Vector3 currentActualRotation;
+        /// <summary>
+        /// The current direction the rotation is moving in.
+        /// </summary>
+        protected float currentRotationDirection;
 
         /// <summary>
         /// Calculates the location of the rotational hinge for the drive.
         /// </summary>
         /// <param name="newHingeLocation">The new local space for the hinge point.</param>
         public abstract void CalculateHingeLocation(Vector3 newHingeLocation);
+        /// <summary>
+        /// Attempts to apply the existing angular velocity back on to the rotation of the drive.
+        /// </summary>
+        /// <param name="multiplier">Multiplies the amount of angular velocity to be applied.</param>
```

Comment:
`The amount to multiply the angular velocity to be applied by.`

Or something... Hard to get this right 😄